src/v2/view-builder/components/ChallengeWebauthnFooter.js

```diff
@@ -0,0 +1,98 @@
+import AuthenticatorFooter from './AuthenticatorFooter';
```

Comment:
All source files require legal headers - let's make sure we add them to all new files please!